srcpkgs/efitools/template

```diff
@@ -0,0 +1,51 @@
+# Template file for 'efitools'
+pkgname=efitools
+version=1.9.2
+revision=1
+build_style=gnu-makefile
+hostmakedepends="perl-File-Slurp"
+makedepends="gnu-efi-libs libressl-devel"
+short_desc="Tools to manipulate EFI secure boot platforms"
+maintainer="Doan Tran Cong Danh <congdanhqx@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://git.kernel.org/pub/scm/linux/kernel/git/jejb/efitools.git"
+distfiles="https://git.kernel.org/pub/scm/linux/kernel/git/jejb/efitools.git/snapshot/${pkgname}-${version}.tar.gz"
+checksum=0f315b36e7d1ba74bfc97ab9f304f0a3072c47578bbe5e42594acae381f9acfe
+patch_args="-Nup1"
+
+subpackages="efitools-efi"
```

Comment:
you don't need to set subpackages manually if the subpackage is build always. This is only needed for packages that not always build all subpackages.